packages/formdata-event/ts_src/Environment/Event.ts

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
I don't consider this blocking FYI, can think more about it later.

Reply:
My only hesitation here is that I like naming files that are strictly about / create one thing with a specific existing name using that exact name. There's the [Google JS style guide](https://google.github.io/styleguide/jsguide.html#file-name) but is there a TS one? There's https://github.com/google/gts, but even though it calls itself a style guide, it doesn't seem to actually contain a human-readable style guide and is more of a tools bundle. (I'm considering adding it here anyway since this project needs a formatter.)

Reply:
The Google TS style guide inherits from the JS one. The TS one doesn't seem to be available publicly, but the internal one has the same rule.

Reply:
Ok, updated.